batch7crud-roll5/src/main/java/com/lftechnology/batch7crud/controller/EmployeeController.java

```diff
@@ -0,0 +1,199 @@
+package com.lftechnology.batch7crud.controller;
+
+import com.lftechnology.batch7crud.model.Employee;
+import com.lftechnology.batch7crud.services.EmployeeService;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Servlet implementation class EmployeeController
+ */
+
+@WebServlet("/employees/*")
+public class EmployeeController extends HttpServlet {
+    private static final long serialVersionUID = 1L;
+    
+    EmployeeService employeeService = new EmployeeService();
+    Employee emp = new Employee();
+    public static String employeeListUrl = "/batch7crud-roll5/employees";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make employeeListUrl a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Make this "public static employeeListUrl" field final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1444)